nodejs-base/ubuntu20.04-node14/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:ubuntu20.04
```

Comment:
What does this do?

Reply:
Created a new ubuntu image with an updated OpenSSL -- now extending the base images from the new ubuntu image.

Reply:
No it's not.

Reply:
Or rather, it's doing that accidentally. The point is, you're now reusing a public name, `ubuntu`, this does not get published in the docker hub for `ubuntu`. This is very unclear and very prone to cause issues.